internal/service/enrollmentrequest.go

```diff
@@ -190,6 +190,20 @@ func (h *ServiceHandler) createDeviceFromEnrollmentRequest(ctx context.Context,
 					Info:   lo.ToPtr(tpmErrorMsg),
 				},
 			}
+		} else {
+			// Device enrolled without TPM - integrity verification not supported
+			deviceStatus.Integrity = api.DeviceIntegrityStatus{
+				Status:       api.DeviceIntegrityStatusUnsupported,
+				Info:         lo.ToPtr("TPM not present or not enabled on this device"),
+				DeviceIdentity: &api.DeviceIntegrityCheckStatus{
+					Status: api.DeviceIntegrityCheckStatusUnsupported,
+					Info:   lo.ToPtr("Device does not have TPM enabled"),
```

Comment:
since we have the Info in the parent DeviceIntegrityStatus just wondering if the children need to echo